### DISCUSSION TITLE
Localized routes with i18n?

### DISCUSSION THREAD

**Discussion by simplenotezy** [vercel/next.js#24870] [Help]
How can you localize routes with next? I'm aware that you can have dynamic parameters, but can you also have "dynamic paths"? E.g.

    /product/[slug] <!-- english
    /produkt/[slug] <!-- german/danish

I haven't really been able to find anything through the documentation and google.

**Comment by bobarros** [comment]
Just create the two pastes with a file [slug] handling the results. 
This is what I did in this project here:

> https://neymar-new.ideias.dev/en 

If you check, every article has one different url.

If you're using getStaticProps with fallback false, you create only the paths you want on each folder, no problem.

If you're using fallback true or blocking, you have to make sure you don't end up generating wrong combinations without realizing it.
